test/buy-tshirt.spec.ts

```diff
@@ -4,9 +4,6 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
   ShippingStepPage, OrderSummaryPage, BankPaymentPage, PaymentStepPage} from '../src/page';
 
 describe('Buy a t-shirt', () => {
-  beforeEach(() => {
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
-  });
```

Comment:
The sleep on line 18 should be removed.

Reply:
I'm not sure if I erased the correct one because the sleep was in line 19, but the test passed so it has to be it.